src/migrations/1716752850275-test-entity.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class TestEntity1716752850275 implements MigrationInterface {
+  name = 'TestEntity1716752850275';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'CREATE TABLE `test` (`id` int NOT NULL AUTO_INCREMENT, `value` text NOT NULL, PRIMARY KEY (`id`)) ENGINE=InnoDB',
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('DROP TABLE `test`');
+  }
+}
```

Comment:
its not needed, it could be left for now, but don't forge to create the task to delete it in next strint